examples/metajob-example/README.md

```diff
@@ -0,0 +1,458 @@
+Overview
+--------
+Meta Jobs allow VDK users to schedule jobs in a directed acyclic graph.
+This means that jobs can be configured to run only when a set of previous jobs have finished successfully.
+
+In this example we will use the Versatile Data Kit to develop four Data jobs - two of these jobs will read data
+from separate json files, and will subsequently insert the data into Trino tables. The third job will read the data
+inserted by the previous two jobs, and will print the data to the terminal. The fourth Data Job will be a Meta Job
+which will manage the other three and ensure that the third job run only when the previous two finish successfully.
```

Comment:
small typo -> ...the third job runS only when...

Reply:
noted